### PR TITLE
Playbook for dojo-day-1 challenge 5 to install lighttpd

### PR DESCRIPTION
**Pull request by vikashsonu900**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
 # platform-dojo-challenges
 Katacoda challenges for Platform-DOJO
+
+### Challenge 5 
+ Install lighttpd on Ubuntu server in katacoda
+ 
+ ### Execution steps
+ ansible-playbook lighttpd-install.yml
```
Comment (zeusbaba) [inline README.md]: minor: plz wrap this with markdown codeblock
`ansible-playbook lighttpd-install.yml`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,8 @@
 # platform-dojo-challenges
 Katacoda challenges for Platform-DOJO
+
+### Challenge 5 
+ Install lighttpd on Ubuntu server in katacoda
+ 
+ ### Execution steps
```
Comment (zeusbaba) [inline README.md]: minor: 1st space should be removed ;)

**Comment by zeusbaba** [comment]
very good job, well done

**Review by zeusbaba — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (vikashsonu900) [reply README.md]: I have added this. Thank you for highligting.

File: README.md (hunk 2 above)
Comment (vikashsonu900) [reply README.md]: Removed the first space in Execution steps.

**Pull request merged**